test/packages_test.rb

```diff
@@ -366,7 +366,7 @@ def package(properties = {})
 
       expect(Yast::Y2Logger.instance).to receive(:info) do |msg|
         expect(msg).to match(
-          /(transaction\sstatus\s[begin|end]|(locked)?resolvables\s
+          /(transaction\sstatus\s[begin|d]|(locked)?resolvables\s
```

Comment:
This looks strange, is it correct?

Reply:
sadly it looks previous regexp is wrong `[begin|d]` is same as `[bdegin|]` it is list of chars...and `e` `n` is already there.